README.md

```diff
@@ -27,8 +27,8 @@ functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
 hardware utilization. Firecracker has also been integrated in container
 runtimes, for example
-[Kata Containers](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)
-and [Weaveworks Ignite](https://github.com/weaveworks/ignite).
+[Kata Containers](https://github.com/kata-containers/kata-containers)
```

Comment:
Hi @Echo-Head-Wall . Thanks for resubmitting.

Our CI is failing on the commit message format now.
Could you keep the subject line short, eg:
```
doc: update README.md with up-to-date links to projects
```
Please also leave a brief description in the body, like:
```
Update links to Kata Containers and Flintlock projects.
```
Also, please use a real email in the `Signed-off-by` line instead of a noreply Github address.

Lastly, could you rebase on top of the main to get rid of the merge commit in the PR.

Thank you!

Reply:
ci would fail if i don't use the one i commit by ( also putting emails in a repo that can be scraped easy sounds like not a great idea). i guess i'll give up. if some one else whose comfortable exposing there email please feel free to make this same change.

Reply:
@Echo-Head-Wall my apologies. I thought a regular email was a prerequisite for the DCO check, but apparently it isn't (we have prior contributions with Github emails, it just isn't very common). If you address the commit message formatting, we will be happy to accept the PR. Sorry for the confusion!

Reply:
https://github.com/firecracker-microvm/firecracker/pull/4701 
ok ill close this one again cause it was to far for me to amend again but that one should work